recipes/folly/all/patches/0017-std-hash.patch

```diff
@@ -0,0 +1,180 @@
+diff --git a/folly/Uri-inl.h b/folly/Uri-inl.h
```

Comment:
Please, add a PR link to the patch, so it's easier to find in the future.

Reply:
https://github.com/conan-io/conan-center-index/pull/5631#discussion_r638924847